From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bacteriawa <A3167717663@hotmail.com>
Date: Wed, 30 Jul 2025 17:34:01 +0800
Subject: [PATCH] Paper: Fix quick craft

Co-authored by: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
As a part of : Paper (https://github.com/PaperMC/Paper/pull/11831/files#diff-3991edc9191addaab9f9701589f8fb4a05001c6f3c9ce1f51d22001f8de805ca)
Licensed under: MIT (https://github.com/PaperMC/Paper/blob/main/licenses/MIT.md)

diff --git a/net/minecraft/world/entity/LivingEntity.java b/net/minecraft/world/entity/LivingEntity.java
index cef064f6de6ee9a54973c222810ba5a121e3a87c..d417ae74c55d22bcdaed76470f8e0b33cdd4d16d 100644
--- a/net/minecraft/world/entity/LivingEntity.java
+++ b/net/minecraft/world/entity/LivingEntity.java
@@ -3988,11 +3988,6 @@ public abstract class LivingEntity extends Entity implements Attackable, Waypoin
             return null;
         } else {
             double d = this.getEyeY() - 0.3F;
-            // Paper start
-            final ItemStack tmp = stack.copy();
-            stack.setCount(0);
-            stack = tmp;
-            // Paper end
             ItemEntity itemEntity = new ItemEntity(this.level(), this.getX(), d, this.getZ(), stack);
             itemEntity.setPickUpDelay(40);
             if (includeThrower) {
